splink/athena/athena_helpers/athena_base.py

```diff
@@ -13,6 +13,33 @@ def regex_extract_sql(col_name, regex):
     """
 
 
+def datediff_sql(
+    col_name_l,
+    col_name_r,
+    date_threshold,
+    date_metric,
```

Comment:
I take it that the user entering an unexpected date_metric (unexpected for us here being anything other than day, month or year), then it will still calculate that without any issues?

Reply:
Yea, if it is valid in presto it should accept it fine